amp.php

```diff
@@ -73,6 +72,19 @@ function amp_init() {
 	}
 }
 
+/**
+ * Define AMP query var.
+ *
+ * This function must be invoked through the 'after_setup_theme' action to allow
+ * the AMP setting to declare the post types support earlier than plugins/theme.
+ *
+ * @since 0.6
+ */
+function define_query_var() {
```

Comment:
We should add hook docs for this filter here since we're touching it.

It was introduced in 0.3.2 via https://github.com/Automattic/amp-wp/commit/c486a1c62f10385bb32d66641a935b621fc3d821